examples/core-api/checkout.php

```diff
@@ -157,12 +134,60 @@ function closeDialog() {
             }
 
             $(".submit-button").click(function (event) {
-                console.log("SUBMIT");
+                var card = {
+                    "card_number": $(".card-number").val(),
+                    "card_exp_month": $(".card-expiry-month").val(),
+                    "card_exp_year": $(".card-expiry-year").val(),
+                    "card_cvv": $(".card-cvv").val()
+                };
+                
                 event.preventDefault();
                 $(this).attr("disabled", "disabled");
+
+                console.log('1. get token_id');
                 MidtransNew3ds.getCardToken(card, callback);
                 return false;
             });
+
+            // callback functions
+            var callback = {
```

Comment:
better let's rename this var to `getCardTokenCallback` for more descriptive code